app/util/Youtube.scala

```diff
@@ -1,233 +0,0 @@
-package util
-
-import java.io.BufferedInputStream
-import java.net.URL
-import java.time.Duration
-import java.util.Date
-
-import com.google.api.client.http.InputStreamContent
-import com.google.api.services.youtube.model.Video
-import com.gu.contentatom.thrift.Atom
-import com.gu.media.YouTubeClient
-import com.gu.media.logging.Logging
-import model.Platform.Youtube
-import model._
-import play.api.{Configuration, Logger}
-
-import scala.collection.JavaConverters._
-
-
-class YouTubeConfig(config: Configuration) {
-  val YouTubeClient(client, _, contentOwner, allowedChannels, disallowedVideos) = YouTubeClient(config.underlying)
-}
-
-case class YouTubeVideoCategoryApi(config: YouTubeConfig) {
-  def list: List[YouTubeVideoCategory] = {
-    val request = config.client.videoCategories()
-      .list("snippet")
-      .setRegionCode("GB")
-
-    request.execute.getItems.asScala.toList
-      .filter(_.getSnippet.getAssignable)
-      .map(YouTubeVideoCategory.build)
-      .sortBy(_.title)
-  }
-}
-
-case class YouTubeVideoUpdateApi(config: YouTubeConfig) extends Logging {
```

Comment:
this is quite a big diff change - I'm going to assume everything has been moved to one of the traits!